### PR TITLE
Refactor docker images and change commit to stable version

### PR DESCRIPTION
**Pull request by Zeeerooo**

### CODE REVIEW

File: adkintunmobile/server/populate/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ MAINTAINER Boris Romero <boris@niclabs.cl>
 ## Dependencies
 # install apt dependencies
 RUN apt-get update && \
-	apt-get install -y \
+	apt-get install -y --no-install-recommends \
```
Comment (fmontoto) [inline adkintunmobile/server/populate/Dockerfile:9]: :+1:

File: adkintunmobile/server/populate/Dockerfile [hunk 2]
```diff
@@ -38,7 +38,8 @@ RUN apt-get purge -y \
     git \
     libpq-dev \
     make \
-    wget
+    wget && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (fmontoto) [inline adkintunmobile/server/populate/Dockerfile]: I think this last command could be in a different RUN command, but no sure. Your decision

**Comment by fmontoto** [comment]
LGTM

File: adkintunmobile/server/populate/Dockerfile (hunk 2 above)
Comment (Zeeerooo) [inline adkintunmobile/server/populate/Dockerfile]: I agree.

**Pull request merged**